### ISSUE TITLE
Please post configuration to implement the semver tutorial

### ISSUE THREAD

**Issue by irons** [callowayproject/bump-my-version#100]
* bump-my-version version: 0.12.0
* Python version: 3.11.5
* Operating System: macOS 13.6.2, debian 12.2 (latest)

### Description

I am not finding sample configuration to support the workflow described at https://callowayproject.github.io/bump-my-version/tutorials/semantic-versioning-example.html, and I've been unable to recreate it. I'll elaborate in another case because my goals differ slightly from what's described there, but seeing how to do what's described there would have been helpful.

**Comment by hottwaj** [comment]
This issue is a bit of a blocker for new potential users (like me :smile:)

Is this the relevant config (found in unit tests)?
https://github.com/callowayproject/bump-my-version/blob/master/tests/fixtures/pep440.toml

**Comment by irons** [comment]
While overstuffed, this config you've found is a working model of semver pre_labels, once you uncomment line 47.

I didn't end up creating the other ticket I alluded to, because I found a workaround, but this example config highlights my main question about the would-be semver tutorial, which is how in step 5 we can go from `1.0.1-rc2` to `1.0.1`, in a single command which doesn't include any explicit state (such as the undesirable `bump-my-version bump patch --new-version 1.0.1`). 

With this config, the tutorial's syntax of `bump-my-version release` returns `Unknown version part: release`. And running `bump pre_label` when the current version is already an rc returns a stack trace ending with the complaint, `ValueError: The part has already the maximum value among ['final', 'a', 'b', 'rc'] and cannot be bumped.` So one can either `bump patch` to a later version than the release candidate, or specify a version explicitly; I'm not finding a third option.
